src/data/game_traits.hpp

```diff
@@ -38,29 +38,29 @@ struct GameTraits
   static constexpr int tileSize = 8;
   static constexpr int tileSizeSquared = tileSize * tileSize;
 
-  static constexpr int viewPortWidthPx = 320;
-  static constexpr int viewPortHeightPx = 200;
-  static constexpr int viewPortWidthTiles = viewPortWidthPx / tileSize;
-  static constexpr int viewPortHeightTiles = viewPortHeightPx / tileSize;
+  static constexpr int viewportWidthPx = 320;
+  static constexpr int viewportHeightPx = 200;
+  static constexpr int viewportWidthTiles = viewportWidthPx / tileSize;
+  static constexpr int viewportHeightTiles = viewportHeightPx / tileSize;
 
-  static constexpr base::Extents viewPortSize{
-    viewPortWidthPx,
-    viewPortHeightPx};
+  static constexpr base::Extents viewportSize{
+    viewportWidthPx,
+    viewportHeightPx};
 
   // The actual in-game viewport starts with an offset and is further reduced
   // to make room for the HUD. The right hand side features another 8px of
   // black border.
-  static constexpr base::Vec2 inGameViewPortOffset{8, 8};
-  static constexpr base::Extents inGameViewPortSize{
-    GameTraits::viewPortWidthPx - 16,
-    GameTraits::viewPortHeightPx - 8};
+  static constexpr base::Vec2 inGameviewportOffset{8, 8};
```

Comment:
In this case, `ViewPort` had an uppercase `V` because it was in the middle of a longer name. The `V` needs to stay capitalized here. So the correct name should be `inGameViewportOffset`.

In general, the `V` in "viewport" can always stay unchanged. The issue is only about changing the `P` in `iewPort` to lowercase, it doesn't matter if it's `ViewPort` or `viewPort` - the "v" stays as it is, only the "P" needs to change